AirCasting/Map/GoogleMapView.swift

```diff
@@ -299,6 +319,15 @@ struct GoogleMapView: UIViewRepresentable {
             let camera = GMSCameraPosition.camera(withLatitude: lat, longitude: long, zoom: 16)
             mapView.animate(to: camera)
         }
+        
+        // Done for search and follow feature
+        private func dotsPositionShouldChange(for mapView: GMSMapView) {
```

Comment:
These are some leftovers from the development process, right?

Reply:
You mean the comment or the function itself?

Reply:
I mean all of the changes in this file. Are they here on purpose? I made a comment because I thought you left them by accident, since you added a new file with searchAndFollowMap